src/core.js

```diff
@@ -848,7 +853,8 @@ $.extend( $.validator, {
 			// aria-describedby should directly reference the error element
 			if ( describer ) {
 				selector = selector + ", #" + describer
-					.replace( /'/g, "\\'" )
+					// See https://api.jquery.com/category/selectors/
+					.replace( /(!|"|#|\$|%|&|'|\(|\)|\*|\+|,|\.|\/|:|;|<|=|>|\?|@|\[|\\|\]|\^|`|\{|\||\}|~)/g, "\\$1" )
```

Comment:
The regex is used in two different methods. I think using it like this will do the trick (I'm not sure)
```js
this.rCssMeta = /.../;
```